packages/core/src/api/tests/formatConversion/export/testExport.test.ts

```diff
@@ -0,0 +1,75 @@
+import { describe, expect, it } from "vitest";
+
+import { BlockNoteEditor } from "../../../../editor/BlockNoteEditor.js";
+import { UnreachableCaseError } from "../../../../util/typescript.js";
+import { blockToNode } from "../../../nodeConversions/blockToNode.js";
+import { setupTestEditor } from "../../setupTestEditor.js";
+import {
+  TestBlockSchema,
+  TestInlineContentSchema,
+  TestStyleSchema,
+} from "../../testSchema.js";
+import { addIdsToBlock } from "../formatConversionTestUtil.js";
+import { ExportTestCase, getExportTestCases } from "./getExportTestCases.js";
+
+// Test for verifying that exporting blocks to other formats works as expected.
+// Used broadly to ensure each block or set of blocks is correctly converted
+// into different formats.
+const testExportTest = async <
+  ConversionType extends "blocknoteHTML" | "html" | "markdown" | "nodes"
+>(
+  editor: BlockNoteEditor<
+    TestBlockSchema,
+    TestInlineContentSchema,
+    TestStyleSchema
+  >,
+  testCase: ExportTestCase<ConversionType>
+) => {
+  (window as any).__TEST_OPTIONS.mockID = 0;
+  if (testCase.conversionType === "blocknoteHTML") {
+    await expect(
```

Comment:
I think there are two approaches we can take here:
a) run all tests for all formats from a single `testExports` file, like we're doing now
b) colocate these tests (like regular unit tests) with the code we're testing (e.g.: markdown tests should live next to the markdownexporter). This file could then still be very minimal and just import the shared test case.

I think there are good arguments for either ((a) is maybe clearer, but (b) is more consistent with how we organize other unit tests)